pyproject.toml

```diff
@@ -20,7 +20,7 @@ classifiers = [
 ]
 requires = [
     "starlette >=0.12.9,<=0.12.9",
-    "pydantic >=0.32.2,<=0.32.2"
+    "pydantic >=0.32.2,<=1.0.0"
```

Comment:
@tiangolo Any chance we could get either a weaker pinning or <=1.1.0? This would enable use with the new pydantic mypy plugin.

Reply:
Yeah, I just tried, but there are some breaking changes (or even a bug maybe?).

And as we are using some internals not covered by SemVer, it's not really Pydantic's fault to change the stuff that is not expected to be used by others.

I think we'll have to stick to keep checking and updating it after testing it when there are new releases.

Reply:
Yeah, I thought that might be the case (regarding internals).

I'll take a look now and see what's breaking v1.1, and take responsibility for opening a pydantic issue (or issues) if necessary.